### DISCUSSION TITLE
link to tasks

### DISCUSSION THREAD

**Discussion by dezren39** [srid/emanote#546] [Ideas]
related: https://github.com/srid/emanote/discussions/105 https://github.com/srid/emanote/discussions/50

i would like if
- i could link to specific tasks within a page
  - tasks automatically get a '#' url heading?
  - if each task getting a heading is too much maybe each list within a page could get a heading?
  - not sure how to make links/embeds that can handle changes to task description or number of tasks in the list.
  	- not a problem in auto-generated indexes
  	- linking in pages or externally, it poses some challenges.
- the tasks index would open to the exact task on-click
  - configurable to disable that if people don't want that?
  - configurable to open in new page?
  - would make the appearance of 'interactivity' actually do something until https://github.com/srid/emanote/discussions/84 allows for other possibilities

in short term i may look at making my task index open the page in general on-click so at least it doesn't do nothing. in documents with large task lists, needing to scroll up-up-up until you find the heading is a bit annoying. an alternate to making the whole thing clickable i guess, would be to put the page name right-aligned on each task as a link.
